spec/order_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'order'
+
+describe Order do
+
+  subject(:order) { described_class.new(menu_klass = menu, order_summary_klass\
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.